src/scenekit.cs

```diff
@@ -121,6 +121,11 @@ interface SCNAnimatable {
 		[Mac (10,10)]
 		[Export ("removeAnimationForKey:fadeOutDuration:")]
 		void RemoveAnimation (NSString key, nfloat duration);
+
+		[TV (10, 0), Mac (10, 12), iOS (10, 0)]
+		[Abstract]
```

Comment:
^ the API diff spotted this one (that I missed earlier). Adding `abstract` members is a breaking change (in .net) and likely not a real, *hard* requirement for ObjC either as existing applications won't have the member defined.

IIRC we simply let them be `optional` with a comment why (and a note in the xtro ignore data files).